c_sharp_json_diff/IntDiffProcessor.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace c_sharp_json_diff
+{
+    class IntDiffProcessor : DiffProcessor
+    {
+
+        public override Dictionary<string, object> PerformDiffProcess(JToken left, JToken right)
+        {
+            string leftValueAsString = left.ToString();
+            string rightValueAsString = right.ToString();
+            NumericDiff diff = new NumericDiff();
+
+            if (int.TryParse(leftValueAsString, out int leftValueAsInt) && int.TryParse(rightValueAsString, out int rightValueAsInt))
```

Comment:
Add property to this similar to the ``DoubleDiffProcessor``'s ``IntProcessing`` property I proposed in another comment. This could have two values:

1. PerfectDivisionOnly - It will only process if both are ints and left perfectly divides into right
2. BothInts - It will process if both are ints

Reply:
As discussed, removed int processor, so this is no longer required